apps/mobile/app/_layout.tsx

```diff
@@ -15,22 +15,25 @@ export const unstable_settings = {
 }
 
 export default function HomeLayout() {
-  const pathname = usePathname()
-  const router = useRouter()
+  const navigation = useNavigationContainerRef()
 
   useEffect(() => {
-    const route = storage.getString('navRoute')
-    if (route && route !== '/') {
-      setTimeout(() => {
-        router.push(route)
-      }, 500)
+    if (__DEV__) {
```

Comment:
why only dev mode? this is very useful in production as well so users can get back to the same location when they reopen the app

Reply:
good catch! others will be developing using Rise production build, so this is very helpful!